provelo_custom/report/hr_employee_overtime_report.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <data>
```

Comment:
You can find [here](https://github.com/OCA/odoo-community.org/blob/master/website/Contribution/CONTRIBUTING.rst#252security-view-and-action) the conventions for naming xml_ids : 

- `act_employee_overtime_report` => hr_employee_action_overtime
- `view_overtime_pivot` => `hr_employee_view_pivot`
- `view_filter_employee_overtime_report` => `hr_employee_view_search`. That xml_id exists so either
    - either overloading the existing one
    - or name it `hr_employee_report_view_search`
    - or leave `hr_employee_menu` since the module name will prevent collision.
- `menu_employee_overtime_report` => `hr_employee_menu`

Reply:
corrected